lib/features/add_bedroom/view/add_bedroom_counter.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:flutter/material.dart';
+import 'package:hatspace/gen/assets.gen.dart';
+import 'package:hatspace/theme/widgets/hs_buttons.dart';
+
+class AddBedroomCounter extends StatelessWidget {
+  final ValueNotifier<int>? counter;
+  final String? text;
+  const AddBedroomCounter(
+      {super.key, required this.counter, required this.text});
+
+  bool checkCounterAdded(int counter) {
+    return counter > 0;
+  }
+
+  bool checkCounterMaxReached(int counter) {
+    return counter == 10;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.only(bottom: 24),
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        crossAxisAlignment: CrossAxisAlignment.center,
+        children: [
+          Text(
+            text!,
```

Comment:
without null check, app may crash here if `text` value is null

Reply:
I removed the nullables so a check here would not be necessary I believe